### PR TITLE
Update core

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo-modules/issues/879 https://github.com/neo-project/neo-modules/issues/880
Require https://github.com/neo-project/neo/pull/3161

### CODE REVIEW

File: src/RpcServer/RpcServer.Wallet.cs [hunk 1]
```diff
@@ -126,10 +126,12 @@ protected virtual JToken ImportPrivKey(JArray _params)
...
         protected virtual JToken CalculateNetworkFee(JArray _params)
         {
-            byte[] tx = Convert.FromBase64String(_params[0].AsString());
+            var tx = Convert.FromBase64String(_params[0].AsString());
 
             JObject account = new();
-            long networkfee = (wallet ?? new DummyWallet(system.Settings)).CalculateNetworkFee(system.StoreView, tx.AsSerializable<Transaction>());
+            var networkfee = Wallets.Helper.CalculateNetworkFee(
+                tx.AsSerializable<Transaction>(), system.StoreView, system.Settings,
+                wallet is not null ? a => wallet?.GetAccount(a).Contract.Script : null);
```
Comment (shargon) [inline src/RpcServer/RpcServer.Wallet.cs:134]:
```suggestion
                wallet is not null ? a => wallet.GetAccount(a).Contract.Script : null);
```

**Review by Jim8y — APPROVED** [review]

**Review by Jim8y — APPROVED** [review]

**Comment by vncoelho** [comment]
let me test

**Comment by shargon** [comment]
> ```cs
> #40 6.697 /opt/neo-modules/src/RpcServer/Utility.cs(38,34): error CS1061: 'NativeContract' does not contain a definition for 'GetContractState' and no accessible extension method 'GetContractState' accepting a first argument of type 'NativeContract' could be found (are you missing a using directive or an assembly reference?) [/opt/neo-modules/src/RpcServer/RpcServer.csproj]
> ```

You don't have updated your submodule

**Comment by shargon** [comment]
> mine is updated

No, is not xD, check the workflow

**Comment by vncoelho** [comment]
My setup is different @shargon 

I am obtaining the last neo locally and replacing the folder entirely

```yaml
# Now it is submodule deps - Removing the existing directory
RUN rm -r /opt/neo-modules/neo
# Copying the new library into place
RUN cp -r /opt/neoLib /opt/neo-modules/neo
# Getting the git version of the module
RUN cd /opt/neoLib && echo "Current neoLib last commit is: $(git rev-parse HEAD)"
# Getting the git version of the submodule
RUN cd /opt/neo-modules && echo "Current neo-modules last commit is: $(git rev-parse HEAD)"
```

**Comment by vncoelho** [comment]
In fact these two were minutes ago....aehauheauea

**Review by vncoelho — APPROVED** [review]

**Pull request merged**